README.md

```diff
@@ -64,40 +65,115 @@ Flags:
+  ```
+
+* Checkout `rosetta` tag `v0.7.6` (`git fetch --all && git checkout v0.7.6`) (or latest released tag) and `make gen-contracts && make all`
+* Run `make alfajores-env` to create an empty datadir with the genesis block (only needs to be run the first time, upon initializing the service)
+
+Then run:
+
+```bash
+go run main.go run \
+  --geth.genesis ./envs/alfajores/genesis.json \
+  --geth.binary ../celo-blockchain/build/bin/geth \
+  --geth.syncmode full \
+  --geth.gcmode archive \
+  --geth.staticnodes "enode://05977f6b7d3e16a99d27b714f8a029a006e41ec7732167d373dd920d31f72b3a1776650798d8763560854369d36867e9564dad13b4b60a90c347feeb491d83a9@34.83.42.50:30303" \
+  --datadir "./envs/alfajores"
+```
 
-## Running Rosetta Docker Image
+You can stop the service and restart by re-running just the last command above (`go run main.go` ... )
 
-Rosetta is released as a docker image: `us.gcr.io/celo-testnet/rosetta`. All version can be found on the [registry page](https://us.gcr.io/celo-testnet/rosetta)
+#### Running on RC1
 
-Within the docker image, we pack `rosetta` binary and also `geth` binary from celo-blockchain. Rosetta will run both.
+This is the same as above with a few differences (generally: specifying `rc1` vs. `alfajores`)
 
-To run Rosetta using the docker container, the following options must be configured:
-  * `genesis.json` for the target network (can be found by `curl 'https://storage.googleapis.com/genesis_blocks/baklava' > genesis.json`)
-  * `staticNodes` or `bootnodes`.
-    * With `staticNodes` Rosetta will directly peer to the list of staticNode provided. This node can be any you have access to. For a public list check `https://storage.cloud.google.com/static_nodes/baklava`
+Prerequisites:
 
-Additionaly, it needs a data directory for the geth datadir & rosetta.db
+* Checkout `celo-monorepo` branch `rc1` instead of `alfajores`, run `yarn && yarn build --ignore docs` as above
+* `celo-blockchain`: same as above
+* Export paths: same as above
+* Checkout `rosetta`: same as above
+* Run `make rc1-env` to create an empty datadir with the genesis block
 
-To run Celo Rosetta:
+Then run:
 
 ```bash
-# Use the last release
-export RELEASE="0.5.4" #might be outdated
+go run main.go run \
+  --geth.genesis ./envs/rc1/genesis.json \
+  --geth.binary ../celo-blockchain/build/bin/geth \
+  --geth.syncmode full \
+  --geth.gcmode archive \
+  --geth.staticnodes "enode://5e0f4e3aaa096e2a2db76622b335cab4d3224d08d16cb11e8855a3a5f30c19d35d81a74b21271562e459495ab203c2f3a5a5747a83eb53ba046aeeb09aa240ff@34.83.110.24:30303" \
+  --datadir "./envs/rc1"
+```
+
+### Version 2: Running Rosetta Docker Image
+
+Rosetta is released as a docker image: `us.gcr.io/celo-testnet/rosetta`. All versions can be found on the [registry page](https://us.gcr.io/celo-testnet/rosetta). Within the docker image, we pack the `rosetta` binary and also the `geth` binary from `celo-blockchain`. Rosetta will run both.
+
```

Comment:
#### Prerequisites:
* [Install](https://docs.docker.com/get-docker/) and run `docker`

Reply:
✅ and added the version that i've tested with